### PR TITLE
[RTL] actctx: When probing a dll for a manifest, use the first manife…

### PR DESCRIPTION
**Pull request by learn-more**
…st id present,

instead of always searching for CREATEPROCESS_MANIFEST_RESOURCE_ID.
CORE-9519

JIRA issue: [CORE-9519](https://jira.reactos.org/browse/CORE-9519)

### CODE REVIEW

File: sdk/lib/rtl/actctx.c [hunk 1]
```diff
@@ -2958,8 +3017,16 @@ static NTSTATUS get_manifest_in_pe_file( struct actctx_loader* acl, struct assem
     NTSTATUS            status;
     SIZE_T              count;
     void               *base;
+    WCHAR resnameBuf[20];
+    LPCWSTR resptr = resname;
 
-    DPRINT( "looking for res %S in %S\n", resname, filename );
+    if ((!((ULONG_PTR)resname >> 16)))
```
Comment (binarymaster) [inline sdk/lib/rtl/actctx.c]: Too many parentheses here. :thinking: 

Also, this way to check string pointer vs. identifier looks a bit hacky.

Comment (HBelusca) [reply sdk/lib/rtl/actctx.c]: There is the `IS_INTRESOURCE` that does just that.

Comment (yagoulas) [reply sdk/lib/rtl/actctx.c]: Which is defined in winuser.h. i doubt we can use this header in ntdll

Comment (HBelusca) [reply sdk/lib/rtl/actctx.c]: You're correct. I was actually just pointing to binarymaster that checking string pointer vs. identifier that way is what windows does.

**Pull request merged**